libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.m

```diff
@@ -26,6 +26,7 @@
 #import "SFRestAPI+Internal.h"
 #import "SalesforceSDKConstants.h"
 #import "SFJsonUtils.h"
+#import "SalesforceSDKCore.h"
```

Comment:
Yeah, probably? I don't know what the conventions are regarding importing umbrella header within the project.